tests/end2end-tests/data_gen/movielens_text.json

```diff
@@ -0,0 +1,54 @@
+{
+    "nodes": [
+            {
+                    "node_id_col":  "id",
+                    "node_type":    "user",
+                    "format":       {"name": "hdf5"},
+                    "files":        "/data/ml-100k/user.hdf5",
+                    "features":     [
+                           {
+                                   "feature_col":  "feat"
+                           }
+                    ]
+            },
+            {
+                    "node_id_col":  "id",
+                    "node_type":    "movie",
+                    "format":       {"name": "parquet"},
+                    "files":        "/data/ml-100k/movie.parquet",
+                    "features":     [
+                           {
+                                   "feature_col":  "title",
+                                   "transform":    {
+                                           "name": "tokenize_hf",
+                                           "bert_model": "bert-base-uncased",
+                                           "max_seq_length": 128
+                                   }
+                           }
+                    ],
+                    "labels":	[
+                        {
+                            "label_col":	"label",
+                            "task_type":	"classification",
+                            "split_pct":	[0.8, 0.1, 0.1]
+                        }
+                    ]
+            }
+    ],
+    "edges": [
```

Comment:
Are you using the same graph for node classification and edge claasification?

Reply:
Yes